polymod/hscript/_internal/PolymodInterpEx.hx

```diff
@@ -322,9 +322,9 @@ class PolymodInterpEx extends Interp
 									for (imp in _proxy._c.imports)
 									{
 										if (imp.name != id0) continue;
-										var finals = PolymodFinalMacro.getAllFinals().get(imp.fullPath) ?? [];
+										var finals = PolymodFinalMacro.getAllFinals().get(imp.fullPath);
```

Comment:
I prefer the original implementation of this line as it leaves `finals` as a non-nullable value.